lib/pbench/client/oidc_admin.py

```diff
@@ -12,7 +12,7 @@ class OIDCAdmin(Connection):
     ADMIN_PASSWORD = os.getenv("OIDC_ADMIN_PASSWORD", "admin")
 
     def __init__(self, server_url: str):
-        super().__init__(server_url, verify=False)
+        super().__init__(server_url, verify=os.environ.get("CURL_CA_BUNDLE", False))
```

Comment:
What's the `False`? You don't seem to have changed the `Connection` constructor, which has `server_url, headers, verify` ... you're defaulting headers by skipping that, and supplying `verify` by name ... what's `False`??

Reply:
`False` is a default value of verify if the `CURL_CA_BUNDLE` env variable is absent. This is a client code to connect to the Keycloak instance and since we are using our custom CA for configuring Keycloak certs I am using this env variable which will override the default the lookup.

Reply:
Yikes: too many parens, and I lost the nesting. Sorry.